public/js/validations.js

```diff
@@ -1,5 +1,5 @@
 function isDate(date) {
-    let pattern = /([12]\d{3})-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01])/;
+    let pattern = /([12]\d{3}-(0[1-9]|1[0-2])-(0[1-9]|[12]\d|3[01]))/;
```

Comment:
why?

Reply:
This makes it exactly the same as the BE vallidation